src/ROOTReader.cc

```diff
@@ -76,6 +76,15 @@ namespace podio {
         branch->GetEntry(localEntry);
       }
     }
+    // load the collections containing vector members
+    auto vecmeminfo = collection->vectorMembers();
+    if (vecmeminfo != nullptr) {
+      for (int i = 0, end = vecmeminfo->size(); i!=end; ++i){
+        branch = m_chain->GetBranch((name+"_"+std::to_string(i)).c_str());
+        branch->SetAddress((*vecmeminfo)[i].second);
+        branch->GetEntry(m_eventNumber);
```

Comment:
`GetEntry` must receive an event number relative to the current `TTree` being processed, otherwise it may crash.

```suggestion
        branch->GetEntry(localEntry);
```